WebDriverAgentTests/IntegrationTests/XCUIElementFBFindTests.m

```diff
@@ -66,8 +66,8 @@ - (void)testSingleDescendantWithClassName
 - (void)testDescendantsWithIdentifier
 {
   NSArray<XCUIElement *> *matchingSnapshots = [self.testedView fb_descendantsMatchingIdentifier:@"Alerts" shouldReturnAfterFirstMatch:NO];
-  XCTAssertEqual(matchingSnapshots.count, 1);
-  XCTAssertEqual(matchingSnapshots.lastObject.elementType, XCUIElementTypeButton);
+  XCTAssertEqual(matchingSnapshots.count, 2);
+  XCTAssertEqual(matchingSnapshots.lastObject.elementType, XCUIElementTypeStaticText);
```

Comment:
iOS13 has different view structure than iOS 12-...?
iOS 12 only has `XCUIElementTypeButton`, but iOS 13 has `XCUIElementTypeButton` and `XCUIElementTypeStaticText` which can match with `Alerts`.
So, the behaviour is expected while we should change the test result for iOS 12- and 13+